README.md

```diff
@@ -1 +1,87 @@
-# baikonur
\ No newline at end of file
+# baikonur
+
+Following instructions are to bootstrap a development environment for running Baikonur
+
+1. Install systemd-nspawn and debootstrap (https://wiki.debian.org/nspawn)
+1. Fetch the ubuntu image for the container: sudo debootstrap focal /var/lib/machines/<name> http://archive.ubuntu.com/ubuntu
+1. Set up the base image for deployment (https://clinta.github.io/getting-started-with-systemd-nspawnd/)
+
+Note: <name> refers to the name of the development container for running Baiknour
+1. Prepare the DNS server
+1. Allow future logins with a preset password
+1. Optionally remove the step below to remove `securetty`. There is a bug which is to be addressed in machinectl, and this is a temporary hack to avoid the annoying lockout after setting up a password in the bootstrap
+1. Optionally allow port forwarding within the container, in case there is no internet access
+```
+rm /var/lib/machines/<name>/etc/hostname \ # Disable the hostname or enter a custom login name
+   && sudo bash -c 'echo "nameserver 8.8.8.8" >> /var/lib/machines/<name>/etc/resolv.conf' \
+   && sudo bash -c 'echo "pts/0" >> /var/lib/machines/<name>/etc/securetty' \
+   && rm /var/lib/machines/<name>/etc/securetty \
+   && sudo bash -c 'echo "net.ip4.forward = 1" >> /var/lib/machines/<name>/etc/sysctl.d/ip_forward.conf'
+```
+1. To deploy use machinectl to start the container: `machinectl start <container-name>`
+
+1. Copy over val-verde source deb packages: `machinectl copy-to <name> /path/to/host/dir /root/path/to/<name>/destination/dir`
+Note: Remove the following packages: 
+`val-verde-libunwind-gnu-haswell.deb 
+val-verde-libunwind-mingw32-haswell.deb 
+val-verde-libcxx-gnu-haswell.deb 
+val-verde-libcxx-mingw32-haswell.deb 
+val-verde-libcxxabi-gnu-haswell.deb 
+val-verde-libcxxabi-mingw32-haswell.deb`
+
+1. Install the dependencies:
+```
+apt update \
+&& apt upgrade -y \
+&& apt install -y libatomic1 libc6-dev libedit2 libmpfr6 libnvidia-compute-440 \
+&& cd /root/path/to/<name>/destination/dir \
+&& dpkg -i /*.deb*
```

Comment:
deb* is superfluous. The packages always end in deb and fixing it will help avoid bugs where an unfortunately named "package.deb.sha1sum" is used as an installation candidate.